progenitor-impl/src/lib.rs

```diff
@@ -487,48 +487,19 @@ impl Generator {
 
     /// Render text output.
     pub fn generate_text(&mut self, spec: &OpenAPI) -> Result<String> {
-        self.generate_text_impl(
-            spec,
-            rustfmt_wrapper::config::Config::default(),
-        )
+        self.generate_text_impl(spec)
     }
 
     /// Render text output and normalize doc comments
-    ///
-    /// Requires a nightly install of `rustfmt` (even if the target project is
-    /// not using nightly).
     pub fn generate_text_normalize_comments(
         &mut self,
         spec: &OpenAPI,
     ) -> Result<String> {
-        self.generate_text_impl(
-            spec,
-            rustfmt_wrapper::config::Config {
-                normalize_doc_attributes: Some(true),
-                wrap_comments: Some(true),
-                ..Default::default()
-            },
-        )
+        self.generate_text_impl(spec)
     }
 
-    fn generate_text_impl(
-        &mut self,
-        spec: &OpenAPI,
-        config: rustfmt_wrapper::config::Config,
-    ) -> Result<String> {
-        let output = self.generate_tokens(spec)?;
-
-        // Format the file with rustfmt.
-        let content = rustfmt_wrapper::rustfmt_config(config, output).unwrap();
-
-        // Add newlines after end-braces at <= two levels of indentation.
-        Ok(if cfg!(not(windows)) {
-            let regex = regex::Regex::new(r#"(})(\n\s{0,8}[^} ])"#).unwrap();
```

Comment:
without the pre-formatting, this regex doesnt work.  
It was a nice touch.
Looks like there is nothing in the rustfmt pipeline to help with this https://rust-lang.github.io/rustfmt/?version=master&search=blank .

An alternative would be to look for pub struct/enum, impl, etc, but attributes make that a bit tricky.

Or maybe this new-line based regex could be provided as a utility function which is demonstrated being used after formatting in the README/example/etc?